meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```
@@ -26,6 +26,12 @@ RDEPENDS:${PN}-dbg = " \
     libgpiod-tools \
     devmem2 \
     libdrm-tests \
+    zarhus-memtools \
+    python3 \
+    heaptrack \
+    gdb \
+    python3-matplotlib \
+    python3-numpy \
```

```suggestion
    gdb \
```

python3 modules are only used by zarhus-memtools and not exactly used by itself for debugging. heaptrack should be rdepended by zarhus-memtools but is also useful by itself so I'm not sure if you want to remove it and treat it as part of `memtools`.

---

https://github.com/zarhus/meta-zarhus/pull/10/files#diff-d4e90c4b34ee683ceff61f6202a43130270b6e245de9038493f2d08b998fae8bR29